mteb/evaluation/MTEB.py

```diff
@@ -497,6 +497,15 @@ def run(
                 del self.tasks[0]  # empty memory
                 continue
 
+            # skip evaluation if the model does not support the task modalities.
+            task_modalities = "".join(sorted(task.metadata.modalities))
+            if "".join(sorted(meta.modalities)) != task_modalities:
```

Comment:
If model support image and text, I think we should run it
```suggestion
            if any([m not in task_modalities for m in meta.modalities])::
```

Reply:
Good call! Will add that.